minidns-core/src/main/java/de/measite/minidns/DNSSECConstants.java

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright 2015 the original author or authors
+ *
+ * This software is licensed under the Apache License, Version 2.0,
+ * the GNU Lesser General Public License version 2 or later ("LGPL")
+ * and the WTFPL.
+ * You may choose either license to govern your use of this software only
+ * upon the condition that you accept all of the terms of either
+ * the Apache License 2.0, the LGPL 2.1+ or the WTFPL.
+ */
+package de.measite.minidns;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public final class DNSSECConstants {
+    /**
+     * Do not allow to instantiate DNSSECConstants
+     */
+    private DNSSECConstants() {
+    }
+
+    /*
+     * DNSSEC Algorithm Numbers
+     * http://www.iana.org/assignments/dns-sec-alg-numbers/dns-sec-alg-numbers.xhtml
+     */
+
+    private static Map<Integer, String> signatureAlogorithmLut;
+
+    static {
+        signatureAlogorithmLut = new HashMap<>();
+        signatureAlogorithmLut.put(1, "RSA/MD5");
+        signatureAlogorithmLut.put(2, "Diffie-Hellman");
+        signatureAlogorithmLut.put(3, "DSA/SHA1");
+        signatureAlogorithmLut.put(5, "RSA/SHA-1");
+        signatureAlogorithmLut.put(6, "DSA-NSEC3-SHA1");
+        signatureAlogorithmLut.put(7, "RSASHA1-NSEC3-SHA1");
+        signatureAlogorithmLut.put(8, "RSA/SHA-256");
+        signatureAlogorithmLut.put(10, "RSA/SHA-512");
+        signatureAlogorithmLut.put(12, "GOST R 34.10-2001");
+        signatureAlogorithmLut.put(13, "ECDSA Curve P-256 with SHA-256");
+        signatureAlogorithmLut.put(14, "ECDSA Curve P-384 with SHA-384");
+    }
+
+    public static String getSignatureAlgorithmName(int algorithm) {
+        if (signatureAlogorithmLut.containsKey(algorithm)) {
+            return signatureAlogorithmLut.get(algorithm);
+        } else {
+            return "Unknown (" + algorithm + ")";
+        }
+    }
+
+    /**
+     * RSA/MD5 (deprecated).
+     */
+    @Deprecated
+    public static final byte SIGNATURE_ALGORITHM_RSAMD5 = 1;
+
```

Comment:
This could be folded into enum classes like DNSMessage.OPCODE (constructor with id + name)